src/pages/components/testimonial/index.js

```diff
@@ -53,19 +58,27 @@ const Testimonial = () => {
           }}
           {...settings}
         >
-          {testimonials.map((item, index) => (
-            <TestimonialCard key={index} item={item} />
-          ))}
+          {testimonials.map(
+            (item, index) =>
+              name !== item.author && (
+                <Link
+                  to={`/testimonial-content/${item.author}`}
+                  target='none'
+                >
+                  <TestimonialCard key={index} item={item} />
+                </Link>
+              )
+          )}
```

Comment:
```suggestion
          {testimonials.map(
            (item, index) =>
              name !== item.author && (
                <Link
                  key={index}
                  to={`/testimonial-content/${item.author}`}
                  target='none'
                >
                  <TestimonialCard item={item} />
                </Link>
              )
          )}
```
There's a console error in key props, pls put the key={index} in the Link component

Reply:
Pls double check if there is still error in the console :bow: